script/check-collections.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const defaultCollections = require('../script/collections/default.json');
+const brandConsolidationCollections = require('../script/collections/brand-consolidation.json');
+
+const checkCollections = buildOptions => (files, metalsmith, done) => {
+  const collection = buildOptions['brand-consolidation-enabled']
+    ? Object.keys(brandConsolidationCollections)
+    : Object.keys(defaultCollections);
+
+  const brokenCollections = Object.keys(files).reduce((accum, key) => {
+    const file = files[key];
+
+    if (key.includes('.html')) {
```

Comment:
You can probably could just `.filter` down to `.html` files right out of the gate. Also, the Path module is considered a more meaningful way of checking the extension.

Reply:
Thanks, that actually would read better.